apps/docs/src/app/platform/component-docs/platform-wizard-generator/platform-wizard-generator-docs.component.html

```diff
@@ -260,3 +260,41 @@
     <fdp-wizard-generator-visibility-between-steps-example></fdp-wizard-generator-visibility-between-steps-example>
 </component-example>
 <code-example [exampleFiles]="fieldsVisibilityExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="special-elements" componentName="wizard-generator"
+    >Special Elements in Wizard Generator
+</fd-docs-section-title>
+<description>
+    <p>This is an example of components which require further attributes rather than just the type declaration.</p>
+    <ul>
+        <li>
+            <b> select </b>: the type <code> select</code> is accompanied by the attribute <code>choices</code> to put
```

Comment:
It would be nice if these docs mentioned that these properties are inherited from functionality that comes from the form generator, and maybe link to relevant documentation there - or provide some examples of each possible type of attribute - there's currently no way for the developer to know what type "choices" really is without digging in to our library code base